omnibus/config/projects/agent.rb

```diff
@@ -94,9 +94,9 @@
   end
   parameters({
     'InstallDir' => install_dir,
-    'InstallFiles' => "#{Omnibus::Config.source_dir()}/datadog-agent/dd-agent/packaging/datadog-agent/win32/install_files",
-    'BinFiles' => "#{Omnibus::Config.source_dir()}/datadog-agent/src/github.com/StackVista/stackstate-agent/bin/agent",
-    'EtcFiles' => "#{Omnibus::Config.source_dir()}\\etc\\datadog-agent",
+    'InstallFiles' => "#{Omnibus::Config.source_dir()}/stackstate-agent/sts-agent/packaging/stackstate-agent/win32/install_files",
```

Comment:
Would be nice if we can clean up that path a bit 😄Seems like xzibit made that directory..   `stackstate-agent/sts-agent/packaging/stackstate-agent` seems a bit overkill

Reply:
![StackStateAgent](https://memecreator.org/static/images/memes/4908941.jpg "StackStateAgent")

Reply:
we can, but that brings ltittle value, because this will not end up in the agent. I would suggest rounding off the windows renaming when we actually go for windows packaging

Reply:
renamed this, should be checked later